dataflow/src/main/java/org/checkerframework/dataflow/cfg/node/ObjectCreationNode.java

```diff
@@ -17,9 +17,15 @@
  * A node for new object creation.
  *
  * <pre>
- *   <em>new constructor(arg1, arg2, ...)</em>
- *   <em>enclosingExpression.new constructor(arg1, arg2, ...)</em>
+ *   <em>new identifier(arg1, arg2, ...)</em>
+ *   <em>new identifier&lt;T&gt;(arg1, arg2, ...)</em>
+ *   <em>enclosingExpression.new identifier(arg1, arg2, ...)</em>
```

Comment:
Do you need to change the `identifier` in this comment to `typeToInstantiate` as well?

Reply:
I think maybe identifier is better here?